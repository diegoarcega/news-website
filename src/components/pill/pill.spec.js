import React from 'react'
import Pill from './pill'
import { render } from '@testing-library/react'

it('renders the pill', () => {
  const { asFragment } = render(<Pill>I'm a text</Pill>)
  expect(asFragment()).toMatchSnapshot()
})
