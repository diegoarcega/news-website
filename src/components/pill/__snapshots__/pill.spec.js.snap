// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the pill 1`] = `
<DocumentFragment>
  .c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  background-color: #fff;
  padding: 10px 25px;
  border-radius: 20px;
  -webkit-transition: background-color ease-in-out 0.2s;
  transition: background-color ease-in-out 0.2s;
}

.c0:hover {
  cursor: pointer;
}

<span
    class="c0"
  >
    I'm a text
  </span>
</DocumentFragment>
`;
